scwx-qt/source/scwx/qt/settings/settings_interface.cpp

```diff
@@ -167,20 +173,24 @@ void SettingsInterface<T>::SetEditWidget(QWidget* widget)
    {
       if constexpr (std::is_same_v<T, std::string>)
       {
-         QObject::connect(hotkeyEdit,
-                          &ui::HotkeyEdit::KeySequenceChanged,
-                          p->context_.get(),
-                          [this](const QKeySequence& sequence)
-                          {
-                             std::string value {
-                                sequence.toString().toStdString()};
+         QObject::connect(
+            hotkeyEdit,
+            &ui::HotkeyEdit::KeySequenceChanged,
+            p->context_.get(),
+            [this, hotkeyEdit](const QKeySequence& sequence)
+            {
+               const std::string value {sequence.toString().toStdString()};
 
-                             // Attempt to stage the value
-                             p->stagedValid_ = p->variable_->StageValue(value);
-                             p->UpdateResetButton();
+               // Attempt to stage the value
+               p->stagedValid_ = p->variable_->StageValue(value);
+               p->UpdateResetButton();
 
-                             // TODO: Display invalid status
-                          });
+               hotkeyEdit->setStyleSheet(p->stagedValid_ ? kValidStyleSheet_ :
+                                                           kInvalidStyleSheet_);
+               hotkeyEdit->setToolTip(p->invalidTooltip_ && !p->stagedValid_ ?
+                                         p->invalidTooltip_->c_str() :
+                                         "");
```

Comment:
I like this! Minor suggestion to refactor these lines into another function, using the `editWidget_` member, to prevent repetition.